examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs

```diff
@@ -483,15 +485,14 @@ public MutateOperation CreateRoot()
         /// <param name="client">The Google Ads client.</param>
         /// <param name="customerId">The Google Ads customer ID.</param>
         /// <param name="assetGroupId">The asset group id for the Performance Max campaign.</param>
-        /// <param name="shouldReplaceExistingProductTree">Option to remove existing product tree
+        /// <param name="replaceExistingTree">Option to remove existing product tree
         /// from the passed in asset group.</param>
         public void Run(
             GoogleAdsClient client,
             long customerId,
             long assetGroupId,
-            bool shouldReplaceExistingProductTree)
+            bool replaceExistingTree)
         {
-            // [START add_performance_max_product_listing_group_tree_1]
             GoogleAdsServiceClient googleAdsServiceClient =
                 client.GetService(Services.V10.GoogleAdsService);
 
```

Comment:
The comment starting on line 501 should be updated. That factory is specifically for create operations, and there is a separate factory for remove operations. In addition, you may want to consider swapping the instantiation of the createFactory with the replaceExistingTree block for a better flow. You don't actually use the create factory until after that block.